### DISCUSSION TITLE
s905x4 TOX3

### DISCUSSION THREAD

**Discussion by x3emal** [ophub/amlogic-s9xxx-armbian#2067] [Amlogic]
Hello can you add support for s905x4 processor plz

**Comment by freezedi** [comment]
yeap is needed. s905x4 more powerful. tv box name x96 max + ultra. EMULELEC 4.6 working with ethernet, but cant find armbian, debian...

**Reply by Zingisa1** [reply]
Is there still no support for s905x4?
